moveit_ros/trajectory_cache/test/cache_insert_policies/test_best_seen_execution_time_policy_with_move_group.cpp

```diff
@@ -424,6 +425,7 @@ TEST_F(MoveGroupFixture, CartesianBestSeenExecutionTimePolicyWorks)
       ASSERT_EQ(*feature_fetch[i], *policy_fetch[i]);
 
       auto longer_plan = msg_plan_pair.second;
+      longer_plan.solution.joint_trajectory.points.back().time_from_start.sec += 1;
       longer_plan.solution.joint_trajectory.points.back().time_from_start.sec *= 10;
```

Comment:
not really sure what these lines do, but these are a little bit magic numbers

Reply:
I'll add a comment to clarify.

Reply:
3f9584db7faf6e458250d3da7c2fba33f4544c06